### DISCUSSION TITLE
Welcome to Spotify4BigSur Discussions!

### DISCUSSION THREAD

**Discussion by fabiusBile** [fabiusBile/Spotify4BigSur#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by herenickname** [comment]
@fabiusBile, привет! Большое спасибо за проект, с переходом на BigSur только этот виджет спас жизнь :)
Но не мог бы ты сделать виджет немного меньше по высоте и уменьшить размер текста? Уж очень громоздко смотрится.

Для сравнения BigSur'овский Now Playing
![image](https://user-images.githubusercontent.com/1989823/103126958-0acfc680-46a1-11eb-8710-e0af63907423.png)
и твой виджет
![image](https://user-images.githubusercontent.com/1989823/103126928-ea077100-46a0-11eb-9fe7-0b100619990a.png)

**Reply by fabiusBile** [reply]
@ekifox привет! Спасибо за фидбек! Ты не первый кто это просит, думаю сделаю во время праздников)

**Reply by fabiusBile** [reply]
Правда высоту самого виджета изменить не получится - WidgetKit позволяет делать виджеты только трёх фиксированных размеров.

**Comment by fabiusBile** [comment]
@ekifox выпустил небольшой апдейт для виджета - теперь он чуть менее громоздкий + добавил большую версию
